node/_tools/test/parallel/test-child-process-spawnsync-maxbuf.js

```diff
@@ -0,0 +1,65 @@
+// deno-fmt-ignore-file
+// deno-lint-ignore-file
+
+// Copyright Joyent and Node contributors. All rights reserved. MIT license.
+// Taken from Node 18.8.0
+// This file is automatically generated by "node/_tools/setup.ts". Do not modify this file manually
+
+'use strict';
+require('../common');
+
+// This test checks that the maxBuffer option for child_process.spawnSync()
+// works as expected.
+
+const assert = require('assert');
+const spawnSync = require('child_process').spawnSync;
+const { getSystemErrorName } = require('util');
+const msgOut = 'this is stdout';
+const msgOutBuf = Buffer.from(`${msgOut}\n`);
+
+const args = [
+  'eval',
+  `console.log("${msgOut}");`,
+];
+
+// Verify that an error is returned if maxBuffer is surpassed.
+{
+  const ret = spawnSync(Deno.execPath(), args, { maxBuffer: 1 });
```

Comment:
Is there a problem with `process.execPath`? It seems to be implemented.

Reply:
I thought `process` properties weren't working. All good, fixed now.